.travis.yml

```diff
@@ -41,9 +41,10 @@ jobs:
         - yarn lint
     - stage: publish docker image
       install: skip
+      script: skip
```

Comment:
Why is this script phase skipped? I think we should not skip it. This will triggered by cron conditions, so by default it should be skipped on PR-s, but will run periodically on master branch.

Reply:
By default, it was running `yarn test` command, so I thought skipping script would fix that. But here it fixed the build failure, but skipped rest of the steps. Will try another approach.

Reply:
As per travis configuration, deploy will be triggered only by cron.